doc/introduction/operations.rst

```diff
@@ -114,6 +114,24 @@ array([[ 0.92387953+0.j,  0.+0.j ,  0.-0.38268343j,  0.+0.j],
 [ 0.+0.j,  0.+0.38268343j,  0.+0.j,  -0.92387953+0.j]])
 
 
+Matrix to Operator functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autosummary::
+
+    ~pennylane.pauli_decompose
+
+These functions take the matrix representation of an operator and return
+the equivalent native PennyLane operator.
```

Comment:
Out of curiosity is there a way to define a custom native gate set? E.g. one in which Hadamard is contained so this outputs 2.0 * [Hadamard] instead?

Reply:
That's exactly what we have planned in the long run! Ideally all the magic that makes pauli arithmetic can be translated to any discrete basis for operators in general. The idea right now is to get the implementation for one specific case and then figure out how we can abstract it later.